packages/check-time/build

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+pushd /pkg/extra
+mkdir -p $PKG_PATH/bin
+gcc -std=gnu11 -Wall -Werror -o $PKG_PATH/bin/check-time check-time.c
```

Comment:
`-static`. No reason to leave this linking to libc, the kernel api here should be much more predictable and stable.